superset-frontend/package-lock.json

```diff
@@ -7670,28 +7670,28 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": false,
+              "resolved": "",
```

Comment:
Now the resolved is updated to empty string? Still a fresh `npm ci` though...

Reply:
lol